### ISSUE TITLE
Uncaught TypeError: Cannot read property _____ of undefined

### ISSUE THREAD

**Issue by gpsugy** [meteor/tutorials#139]
Following the todo-list tutorial with Meteor and React steps on adding imports/ui/App, I have received many errors in my console that follow the format:

"Uncaught TypeError: Cannot read property ____ of undefined"

Examples that fill in the blank are meteorInstall, Random, MongoID.

I have commented out the code to narrow down when exactly the problem occurs, and it happens only when my code for import/ui/App.jsx is like so:

```
import React, { Component } from 'react';
import { Meteor } from 'meteor/meteor';

export default class App extends Component {
	render() {
		return (
			<div className="container">
			</div>
		)
	}
}
```

When I completely comment out the class component for App so that only the import lines of code exist, however, the errors are removed (so this does not seem to be a problem with transpiling es6 code, since the imports work). Perhaps one of the node module dependencies is broken?

Any help is greatly appreciated.

**Comment by tizbecool** [comment]
I've got the same trouble. I think it's also a problem with transpiling es6 and missing packages into the base project.
I solved this by taking the final package file from the tutorial project repository.
https://github.com/meteor/simple-todos-react and run "meteor npm install" to install all packages and its works.
I'm new in meteor and react so I can't give you the precise reason or the precise missing package or not ^^
